src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -219,4 +218,204 @@ public void getResources() throws Exception {
             .collect(Collectors.toList()),
         Matchers.containsInAnyOrder(id1, id2));
   }
+
+  @Test
+  public void abandonThenBumpResources() throws Exception {
```

Comment:
[optional] I feel like "abandon resource" and "bump resource" are two well-separated pieces of functionality that are better off tested independently. This test is >100 lines long... maybe consider splitting it up into an abandonResource and then a bumpResource test (even if you have to share some setup code)?

Reply:
Done